themes/qgrid/source/js/menu.js

```diff
@@ -1,4 +1,4 @@
-function toggleVisibility(e) {
```

Comment:
what i want to see on high level for the search functionality here:

when user types something in search input:
```javascript
const search = ...;
setHash('search', search);
filterMenu(search); 
highlightMenuItems(search); 
highlightPageText(search);
```
when page is loaded with search
```javascript
const search = getHash('search');
if(search) {
   filterMenu(search); 
   highlightMenuItems(search); 
   highlightPageText(search);
}
```